docs/photoniq/vwrs/05-monitoring-vwrs/retrieve-metrics.md

```diff
@@ -1,38 +1,19 @@
 ---
-sidebar_position: 100
-title: Get VWRs Metrics
+title: Retrieve Sample Metrics
 ---
 
-Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) has built-in metrics that allow you to retrieve waiting room telemetry per waiting room, per minute. Data is stored for the past 90 days.
-
-Using VWRs metrics, you can:
-
-- Evaluate peak traffic flow through your waiting room and onto your site.
-- Determine how long users spent in the waiting room.
-- Use analytics to help calibrate your waiting room settings.
-
 ## Get Metrics
 
-You can retrieve metrics using the following endpoints:
+VWRs offers different endpoint for retrieving your waiting room metrics:
 
 - [Get Metrics for a Domain](https://www.macrometa.com/docs/apiVwrs#/operations/getMetrics) - Retrieve the metrics for a specific domain. Set start and end times to get metrics for a specific time period. To get the domain key, use the [Get Information for All Domains](https://www.macrometa.com/docs/apiVwrs#/operations/getDomainUsingQueryParams) endpoint.
```

Comment:
```suggestion
- [Get Metrics for a Waiting Room](https://www.macrometa.com/docs/apiVwrs#/operations/getMetrics) - Retrieve the metrics for a specific waiting room. Set start and end times to get metrics for a specific time period. To get the waiting room key, use the [Get Information for All Waiting Rooms](https://www.macrometa.com/docs/apiVwrs#/operations/getDomainUsingQueryParams) endpoint.
```

Reply:
@todor-macrometa is this URL correct? (https://www.macrometa.com/docs/apiVwrs#/operations/getDomainUsingQueryParams) 
Did we keep this "Domain" or did we rename it to "WaitingRoom"?

Reply:
@maryann-macrometa @joubert-macrometa 

The correct link is: https://www.macrometa.com/docs/apiVwrs#/operations/getWaitingRoomUsingQueryParams

This has been changed in the `main` branch (and the official docs page), but this branch does not contain the changes.